### DISCUSSION TITLE
Tells me too late that interactions have been restricted

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#4567] [Mobile] [answered]
I was trying to comment on <https://github.com/aspnet/Announcements/pull/468> and got this error:

> could not be created. Interactions on this repository have been restricted to collaborators only.

![image](https://user-images.githubusercontent.com/46201428/124996622-edb58080-e051-11eb-873b-fc3076fa7a29.png)

Two problems with that:

1. Something is missing from the beginning of the error message. What could not be created?

2. If I am not authorised to comment, then the app should tell me about that before I spend time writing the comment.

**Comment by mxie** [comment]
Hi @KalleOlaviNiemitalo - I'm sorry to hear that you're having issues editing your comment. Is this specifically when you're editing an existing comment or adding a new one?

**Reply by KalleOlaviNiemitalo** [reply]
This was for a new comment.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the context in your other comment. I'm going to open an issue on our end to help people avoid situations like this in the future.
